### PR TITLE
feat(axe-core-4.3.2): Upgrade to use axe-core 4.3.2

### PR DESCRIPTION
**Pull request by brocktaylor7**
#### Details

This PR upgrades the axe-sarif converter to use the latest version of axe-core: v4.3.2.

##### Motivation

Feature work, upgrading all products to use axe-core 4.3.2

##### Context

There is currently a bug in @axe-core/puppeteer that causes the standard w3citylights-axe-4.3.2.reporter-raw.json test resource to fail to generate. An issue has been filed with Deque here:

https://github.com/dequelabs/axe-core-npm/issues/323

That file has not been tested against in our past unit tests, so it shouldn't be a blocking issue but may be something we want to revisit once the issue has been resolved. The basic-axe-4.3.2.reporter-raw.json, which _is_ referenced in unit tests did generate properly and passes the tests.

The chromedriver yarn resolution was added due to selenium's chromedriver being out of date and causing issues running the test resource generator. The fact that we are able to use a ">=" version should allow it to continue upgrading into the future without removing the resolution, making this resolution less likely than most to cause issues down the line.

#### Pull request checklist

- [x] PR title respects [Conventional Commits](https://www.conventionalcommits.org) (starts with `fix:`, `feat:`, etc, and is suitable for user-facing release notes)
- [x] PR contains no breaking changes, **OR** description of both PR **and final merge commit** starts with `BREAKING CHANGE:`
- [n/a] (if applicable) Addresses issue: #0000
- [x] Added relevant unit tests for your changes
- [x] Ran `yarn precheckin`
- [x] Verified code coverage for the changes made

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ The [microsoft/axe-pipelines-samples](https://github.com/microsoft/axe-pipelines
 
 The version number of this library is **independent** from the version numbers of the axe-core inputs and SARIF outputs it supports.
 
--   axe-sarif-converter version 2.x supports input from version ^3.2.0 || ^4.0.0 of axe-core (tested with 3.2.2, 3.3.2, 3.4.1, 3.4.2, 3.5.1, 3.5.2, 3.5.3, 3.5.4, 3.5.5, 4.0.1, 4.0.2, 4.1.1, and 4.2.0) and outputs SARIF v2.1
+-   axe-sarif-converter version 2.x supports input from version ^3.2.0 || ^4.0.0 of axe-core (tested with 3.2.2, 3.3.2, 3.4.1, 3.4.2, 3.5.1, 3.5.2, 3.5.3, 3.5.4, 3.5.5, 4.0.1, 4.0.2, 4.1.1, 4.2.0 and 4.3.2) and outputs SARIF v2.1
```
Comment (dbjorge) [inline README.md:69]: nit: [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/punctuation/commas) suggests keeping the Oxford comma

**Review by dbjorge — DISMISSED** [review]
LGTM modulo the nit comment!

File: README.md (hunk 1 above)
Comment (brocktaylor7) [reply README.md:69]: The Brock style guide also strongly suggests this, this was not intentional, but a great catch!

**Review by dbjorge — APPROVED** [review]

**Pull request merged**